### ISSUE TITLE
Vulnerabilities:

### ISSUE THREAD

**Issue by Wjplatformer** [Isaiah08-D/findme#1]
1. No Security Policy
2. No encryption
3. No License
4. No code of conduct
5. No security advisories

**Comment by Isaiah08-D** [comment]
Yeah, not really done. I was thinking more like nonencrypted keys sent through a websocket or SQL Injection Attacks, but these are good additions.

**Comment by LogSkele** [comment]
i thought you didn't finish this lmao
